### ISSUE TITLE
Homepage not saving acceptance

### ISSUE THREAD

**Issue by King-Nerd** [Pauloo27/homepage#3]
Hey Pauloo, love the homepage, but it seems you updated it recently and now, every time I try to open a new tab, it pulls your popup up asking about theme and settings, even after I have accepted multiple times. I don't know if my browser makes a difference, but I use the Brave Browser (built on Chromium).

**Comment by Pauloo27** [comment]
Hello, how are you running it? Some kinda of seft host or using the GitHub version (https://pauloo27.github.io/homepage)?

**Comment by King-Nerd** [comment]
I am just using the Github version

**Comment by Pauloo27** [comment]
1) How often it happens? I also use Brave and I think it only happened to me once or twice.

2) Can you describe how you reproduce it? 

3) Also, can you open the Dev Tools console in the homepage (Ctrl + F12) and run `localStorage.getItem("version")`? It should return `'2.4.0'`

4) Since when you noticed the problem?

5) What Brave version are you in?

**Comment by Pauloo27** [comment]
TL;DR: It's a issue with Brave (https://github.com/brave/brave-browser/issues/17859)

After a few hours testing, I got the same problem.

The thing is that homepage looks in the localStorage for the `version` item, if it doesn't find it, it shows the welcome popup.

With some debug I've found that only "half" the localStorage is loaded when the check is made, and the version is not included in the loaded half resulting in a corrupt state.
